pghoard/rohmu/object_storage/azure.py

```diff
@@ -286,6 +311,9 @@ def progress_callback(bytes_sent, _):
 
     def get_or_create_container(self, container_name):
         start_time = time.monotonic()
-        self.conn.create_container(container_name)
+        try:
+            self.conn.create_container(container_name)
+        except ResourceExistsError:
+            pass  # This was handled by azure-storagepython, with azure-sdk we need to do it ourselves
```

Comment:
I would remove this comment, it won't make much sense after this has been merged and nobody knows some other SDK was even used.